### DISCUSSION TITLE
Disable dropdown of Wishlist in header

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#142] [1. Help] [answered]
Hi,
How to disable wishlist dropdown in header and redirect to wishlist page(in case of logged in user) and login page (In case of guest user).
Currently it's opening a dropdown in header when we click on wishlist link.

**Comment by vovayatsyuk** [comment, marked as answer]
When dropdown content is empty it won't show it. Wishlist dropdown content is added in [blank theme](https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/Magento_Theme/layout/default.xml#L38)

To remove it, use layout update xml in your theme:

```xml
<referenceBlock name="wishlist_sidebar" remove="true"/>
```

**Reply by dheerajKsharma** [reply]
Thanks Much!
